examples/complete/privileged-principals.tf

```diff
@@ -0,0 +1,99 @@
+locals {
+  principal_names = [
+    "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${join("", module.deployment_principal_label.*.id)}",
```

Comment:
make a local from `data.aws_caller_identity.current.account_id` and use it here

Reply:
resolved